src/status_im/ui/screens/chat/styles/main.cljs

```diff
@@ -238,6 +238,29 @@
   {:margin-bottom 5
    :color         colors/black})
 
+(def empty-chat-name
+  {:style {:font-size     22
```

Comment:
could you please move style key outside?

Reply:
done!